core/src/forwarding_stage.rs

```diff
@@ -586,9 +586,9 @@ impl TpuClientNextClient {
         ConnectionWorkersSchedulerConfig {
             bind: BindTarget::Socket(bind_socket),
             stake_identity: stake_identity.map(StakeIdentity::new),
-            // Cache size of 128 covers all nodes above the P90 slot count threshold,
-            // which together account for ~75% of total slots in the epoch.
-            num_connections: 128,
+            // Cache size of 64 covers all nodes for at least next 64 * 4 slots
+            // which corresponds to ~100sec.
```

Comment:
We should probably use the same const with the same description in both instances to avoid confusion in configuration.